File: app-components/src/internalTesting/java/org/synsystems/onlypass/components/InternalTestingApp.java
import org.synsystems.onlypass.components.App;
import org.synsystems.onlypass.components.AppComponent;

import org.synsystems.onlypass.components.DaggerInternalTestingAppComponent;

public class InternalTestingApp extends App {
  @Override
  protected AppComponent createAppComponent() {
    return DaggerInternalTestingAppComponent.create();
  }
}

File: app-components/src/internalTesting/java/org/synsystems/onlypass/components/InternalTestingApp.kt
import org.synsystems.onlypass.components.App
import org.synsystems.onlypass.components.AppComponent
import org.synsystems.onlypass.components.DaggerInternalTestingAppComponent

class InternalTestingApp : App() {

  @Override
  protected fun createAppComponent() = DaggerInternalTestingAppComponent.create()
}